### DISCUSSION TITLE
Would like to be able to add options to configure the source.list

### DISCUSSION THREAD

**Discussion by lslvxy** [tteck/Proxmox#414] [Ideas]

**Comment by tteck** [comment]
`nano /etc/apt/sources.list`

**Reply by lslvxy** [reply]
i mean in scripts we can choose which source to config.

in LXC scripts always use default sources

when run `Updating Container OS...` it  takes too much time

**Comment by lslvxy** [comment]
ustc source :` https://mirrors.ustc.edu.cn/help/ubuntu.html`

tuna source ：` https://mirrors.tuna.tsinghua.edu.cn/`


aliyun source ： `https://developer.aliyun.com/mirror/`
